cudax/include/cuda/experimental/__stf/stream/stream_task.cuh

```diff
@@ -602,7 +602,7 @@ public:
     }
   }
 
-private:
+  // private:
```

Comment:
😲

Reply:
Indeed this is a temporary hack, we need to get rid of that to make the implementation compatible with CUDA graphs anyway.